docs/workforce-management-framework/topics/con-passportjs-module.adoc

```diff
@@ -0,0 +1,17 @@
+[id='con-passportauth-module-{chapter}']
+= Introducing the {WFM-RC-NameShort} Passport.js Module
+
+*Passport.js* is a lightweight authentication middleware for Node.js that is extremely flexible and modular.
+*Passport.js* offers a comprehensive set of link:http://passportjs.org/docs/configure[strategies] that support authentication using a username and password, Facebook, Twitter, and more(?).
```

Comment:
"that supports authentication using a username and password, Facebook, Twitter, and more(?)."

I don't really understand this. It " supports authentication using a username and password". "Facebook, Twitter, and more(?)." - I don't get.